tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -58,16 +76,26 @@ module('Acceptance | guid file | registration files', hooks => {
         assert.dom('[data-test-tags-button]').exists('Tags button exists');
         await percySnapshot('Acceptance | guid file | registration files | mobile view | file renderer');
 
+        assert.dom('[data-test-file-renderer]').isVisible();
+        assert.dom('[data-test-metadata-tab]').isNotVisible();
+        assert.dom('[data-test-revisions-tab]').isNotVisible();
+        assert.dom('[data-test-tags-tab]').isNotVisible();
+
+        // click tags and verify appropriate panels shown
         await click('[data-test-versions-button]');
         assert.dom('[data-test-revisions-tab]').exists('Revisions shown');
-        assert.dom('[data-test-file-renderer]').doesNotExist('File renderer is hidden');
+        assert.dom('[data-test-file-renderer]').isNotVisible('File renderer is hidden');
```

Comment:
Per our discussion in scrum the other week, these should be `.doesNotExist` unless it's just not true (but it will be for all these tests, I believe).

Reply:
That sounds good. This has now been updated.